server/src/lib/model/user-client.ts

```diff
@@ -75,10 +108,24 @@ const UserClient = {
   }) {
     const [rows] = await db.query(
       `
-        SELECT u.*, accents.accent, locales.name AS locale
+        SELECT u.*, accents.accent, locales.name AS locale, ages.age, sexes.sex
         FROM user_clients u
         LEFT JOIN user_client_accents accents on u.client_id = accents.client_id
         LEFT JOIN locales on accents.locale_id = locales.id
+        -- TODO: This subquery is VERY awkward, but safer until we simplify
+                 accent grouping.
+        CROSS JOIN
+          (SELECT ages.id AS age_id, sexes.id AS sex_id
+            FROM user_clients
+            LEFT JOIN demographics ON user_clients.client_id = demographics.client_id
+            WHERE user_clients.${
+              client_id ? `client_id = ${client_id}` : `email = ${email}`
+            }
```

Comment:
This honestly feels like magic haha 🎉

Reply:
Hahah I know 😞 dark, dangerous magic. Leaving a note for my future self below, feel free to disregard.

---

Hi Riley! I hope 2021 is off to a good start. Yes, this is what was causing that horrible bug – sorry.